src/pages/optimized-for-change/index.tr.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Optimized for Change
```

Comment:
Hi, you missed the title and the spoiler

Reply:
Oh no, sorry about that, updated now